docs/custom_configuration/index.md

```diff
@@ -0,0 +1,93 @@
+# Custom Configuration
+
+Testcontainers for .NET supports various configurations to set up your test environment. It automatically discovers the Docker environment and applies the configuration. Set or override the default values either with the Testcontainers [properties file][properties-file-format] (`~/testcontainers.properties`) or with environment variables. If you prefer to configure your test environment at runtime, set or override the configuration through the `TestcontainersSettings` class. The following configurations are available:
+
+| Properties File         | Environment Variable                   | Description                                                                                                               |
```

Comment:
We should add the default values to the table (if they make sense).

Reply:
I've added them.